Cargo.toml

```diff
@@ -8,19 +8,15 @@ members = [
     "protocol-units/execution/monza/*",
     "protocol-units/execution/suzuka/*",
     "protocol-units/da/m1/*",
-    "protocol-units/sequencing/*",
+    "protocol-units/sequencing/memseq/*",
     "protocol-units/mempool/*",
     "protocol-units/settlement/mcr/client",
     "protocol-units/settlement/mcr/manager",
     "util/*",
     "util/buildtime/buildtime-helpers",
     "util/buildtime/buildtime-macros",
-    "networks/monza/monza-client",
-    "networks/monza/monza-config",
-    "networks/monza/monza-full-node",
-    "networks/suzuka/suzuka-client",
-    "networks/suzuka/suzuka-config",
-    "networks/suzuka/suzuka-full-node",
+    "networks/monza/*",
+    "networks/suzuka/*",
```

Comment:
OK, as long as all relevant crates are listed in the workspace dependencies anyway.

Reply:
Yeah, for now, that's what I'd like to strive for up at `networks/`.